jwst/pipeline/calwebb_coron3.cfg

```diff
@@ -4,6 +4,7 @@ class = "jwst.pipeline.Coron3Pipeline"
     [steps]
       [[stack_refs]]
       [[align_refs]]
-      [[klip]]
+        config_file = align_refs.cfg
+      [[klip]] 
       [[outlier_detection]]
```

Comment:
Something I missed in the related moving of the outlier detection PR: The `[[outlier_detection]]` step should now be the first step. This doesn't affect operations, but is just for consistency with order.

Reply:
Moved this to the first step